src/constraints/transport.jl

```diff
@@ -109,14 +109,14 @@ function _append_transport_data_to_indices(connection)
             ANY_VALUE(cons.time_block_end) AS time_block_end,
             ANY_VALUE(cons.var_flow_id) AS var_flow_id,
             ANY_VALUE(flow.capacity) AS capacity,
-            ARRAY_AGG(expr_avail.id) AS avail_indices,
-            ARRAY_AGG(expr_avail.commission_year) AS avail_commission_years,
+            ANY_VALUE(expr_avail.id) AS avail_indices,
```

Comment:
If there is no aggregation, there should be no need to `GROUP BY` and this can be singular as well.